food_trucks/__manifest__.py

```diff
@@ -0,0 +1,51 @@
+{
+  'name': 'Food Trucks',
+  'version': '1.0',
+  'category': 'Hospitality',
+  'author': 'Odoo S.A.',
+  'depends': [
+      'industry_fsm_sale_report',
+      'knowledge',
+      'planning',
+      'pos_restaurant_appointment',
```

Comment:
I don't think we need appointment
```suggestion
      'pos_restaurant',
```